sdk/communication/communication-administration/review/communication-administration.api.md

```diff
@@ -42,7 +44,16 @@ export interface AreaCodes {
 export type AssignmentStatus = "Unassigned" | "Unknown" | "UserAssigned" | "ConferenceAssigned" | "FirstPartyAppAssigned" | "ThirdPartyAppAssigned";
```

Comment:
I still see a lot of generated model types where all properties are optional. Didn't this get fixed recently? @whisper6284